bazel/boost.bzl

```diff
@@ -13,6 +13,10 @@ def boost_library(name, deps = []):
                     "libboost_{}.a".format(name),
                     "libboost_{}.dylib".format(name)
                 ],
+                "clang": [
```

Comment:
Do we need it here? This seems the same as `//conditions:default`.

Reply:
there's a difference, it adds `toolset=clang`. without this, boost compilation fails, https://github.com/istio-ecosystem/authservice/issues/155